src/workerd/server/workerd.c++

```diff
@@ -927,6 +929,12 @@ public:
     server.enableControl(fd);
   }
 
+  void diskCacheDir(kj::StringPtr pathStr) {
+    kj::Path path = fs->getCurrentPath().eval(pathStr);
+    kj::Maybe<kj::Own<const kj::Directory>> dir = fs->getRoot().tryOpenSubdir(path, kj::WriteMode::MODIFY);
+    server.setDiskCacheRoot(kj::mv(dir));
+  }
```

Comment:
Should this parameter have a default value pointing at `$TMPDIR/workerd-python-package-cache`?

Reply:
On linux and mac I'd like to see it default to:
* `$XDG_CACHE_HOME/workerd-python-package-cache` if `XDG_CACHE_HOME` is defined
* `$HOME/.cache/workerd-python-package-cache` otherwise.

Unless you have a strong opinion that it should go in `/tmp` instead.

Reply:
True, better to follow the XDG spec here.

For macOS I think we want `~/Library/Caches/` and I guess `%LOCALAPPDATA%` for Windows.